README.md

```diff
@@ -111,17 +117,17 @@ packages:
     - package: dbt-labs/dbt_utils
       version: [">=0.8.0", "<0.9.0"]
 ```
-
 # 🙌 How is this package maintained and can I contribute?
 ## Package Maintenance
-The Fivetran team maintaining this package **only** maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/google_play/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_google_play/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
+The Fivetran team maintaining this package _only_ maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/google_play/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_google_play/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
 
+## Opinionated Decisions
```

Comment:
This package includes a DECISIONLOG.md; however, it is never referenced within the README. I felt this would be a good location to reference the DECISIONLOG.

Would you be able to add that reference here.